### PR TITLE
extend/duplicate .widget-title and .accent-header styles to misspelled class name caused by known bug in Gutenberg to prevent nested legacy widget titles from losing styles

### PR DESCRIPTION
**Pull request by 9ete**

### CODE REVIEW

**Comment by hawkeye126** [comment]
Steps to replicate:
1) Using default WordPress and News Pack Theme (any child theme)
2) Add a widget group
3) Add two legacy widgets of type meta
4) Add widget titles
5) On front end you will see the `.widget-title` and `.accent-header` classes are missing

Before:
Screen Shot 2021-09-30 at 6 06 00 PM

After:
Screen Shot 2021-09-30 at 6 05 38 PM

**Comment by adekbadek** [comment]
Pinging @laurelfulford ;)

**Review by laurelfulford — CHANGES_REQUESTED** [review]
Thank you for this PR, @hawkeye126! 

It looks like updates will need to be made to more places to fix this everywhere -- the Newspack theme is pretty convoluted, between its customization options and different child themes. Unfortunately this means making these kinds of fixes can be pretty intensive! :) 

I've gone through and tested different cases with the different themes, to try to capture most of the cases I'm talking about. You're welcome to update your PR, but of course I understand if this is a bit much! Please let me know if you'd like me to take a swing at this. 

Thank you!

**Newspack (parent theme):**
* The sidebar widget title uses the 'primary' colour for text, so it should change when that's changed under Customizer > Colors - this is added through the `accent-header` class, but it will likely need the `#secondary .widgettitle` selector added to what's there ([ref](https://github.com/Automattic/newspack-theme/blob/master/newspack-theme/inc/color-patterns.php#L347-L350))

image

* The sidebar widget title should only be uppercase when the `Use all-caps for accent text` option is selected under the Customizer > Typography panel  -- like the above, `#secondary .widgettitle` will probably need to be added to what's there ([ref](https://github.com/Automattic/newspack-theme/blob/master/newspack-theme/inc/typography.php#L389-L392)).

image

* In the footer, the widget titles should not have the underline - that's only added to the widgets in the sidebar area:

image

* In the footer, when you set a darker background colour under Customizer > Colors, the widget titles should be white ([ref](https://github.com/Automattic/newspack-theme/blob/master/newspack-theme/inc/color-patterns.php#L303-L309))

image


**Newspack Joseph**
* The sidebar widget title should have a horizontal line next to it ([ref](https://github.com/Automattic/newspack-theme/blob/master/newspack-joseph/sass/style.scss#L117-L147))
* The sidebar widget title should only be uppercase when the `Use all-caps for accent text` option is selected under the Customizer > Typography panel ([ref](https://github.com/Automattic/newspack-theme/blob/master/newspack-joseph/inc/child-typography.php#L76-L79))
image

* In the footer, when you set a darker background colour under Customizer > Colors, the widget titles should be white ([ref](https://github.com/Automattic/newspack-theme/blob/master/newspack-joseph/inc/child-color-patterns.php#L72-L76))

image

**Newspack Katharine**
* Similar to the default theme, the sidebar widget title should pick up the primary colour set under Customizer > Colors ([ref](https://github.com/Automattic/newspack-theme/blob/master/newspack-katharine/inc/child-color-patterns.php#L43-L45))

* The sidebar widget title should only be uppercase when the `Use all-caps for accent text` option is selected under the Customizer > Typography panel ([ref](https://github.com/Automattic/newspack-theme/blob/master/newspack-katharine/inc/child-typography.php#L33-L43))
image

* The footer widget title should also be uppercase when the `Use all-caps for accent text` option is selected under the Customizer > Typography panel ([ref](https://github.com/Automattic/newspack-theme/blob/master/newspack-katharine/inc/child-typography.php#L33-L43))
* In the footer, the widget title should not be picking up the little 'above title' accent
image

**Newspack Nelson**

* The sidebar widget title should only be uppercase when the `Use all-caps for accent text` option is selected under the Customizer > Typography panel ([ref](https://github.com/Automattic/newspack-theme/blob/master/newspack-nelson/inc/child-typography.php#L36-L49))
image

* The footer widget title should also be uppercase when the `Use all-caps for accent text` option is selected under the Customizer > Typography panel ([ref](https://github.com/Automattic/newspack-theme/blob/master/newspack-nelson/inc/child-typography.php#L46-L49))
image

**Newspack Sacha**
* In the sitebar the widget title should pick up the primary colour set under Customizer > Colors ([ref](https://github.com/Automattic/newspack-theme/blob/master/newspack-sacha/inc/child-color-patterns.php#L34-L46))
* The sidebar widget title should only be uppercase when the `Use all-caps for accent text` option is selected under the Customizer > Typography panel ([ref](https://github.com/Automattic/newspack-theme/blob/master/newspack-sacha/inc/child-typography.php#L43-L48))
image
* In the footer, the widget titles should not have the borders to the left and right:
* In the footer, if the background colour is set to 'default' under Customizer > Colors, the widget titles should pick up the primary colour ([ref](https://github.com/Automattic/newspack-theme/blob/master/newspack-sacha/sass/style.scss#L487-L490))
* image
* In the footer, if the background colour is set to custom under Customizer > Colors the widget titles should be white on darker backgrounds, and black on lighter backgrounds ([ref](https://github.com/Automattic/newspack-theme/blob/master/newspack-sacha/inc/child-color-patterns.php#L91-L94))
image


**Newspack Scott**

* The sidebar widget title should only be uppercase when the `Use all-caps for accent text` option is selected under the Customizer > Typography panel ([ref](https://github.com/Automattic/newspack-theme/blob/master/newspack-scott/inc/child-typography.php#L42))
image
* The footer widget title should also be uppercase when the `Use all-caps for accent text` option is selected under the Customizer > Typography panel ([ref](https://github.com/Automattic/newspack-theme/blob/master/newspack-scott/inc/child-typography.php#L44-L46))
image
* In the footer, if the background colour is set to a darker colour under Customizer > Colors the widget titles should be white ([ref](https://github.com/Automattic/newspack-theme/blob/master/newspack-scott/inc/child-color-patterns.php#L101-L105))
image

Again, thank for diving into this! Please let me know if you have any questions about the above!

File: newspack-theme/inc/color-patterns.php [hunk 1]
```diff
@@ -303,7 +303,8 @@ function newspack_custom_colors_css() {
 				.site-footer,
 				.site-footer a,
 				.site-footer a:hover,
-				.site-footer .widget-title,
+				.site-footer .widget-title,,
+				.site-footer .widgettitle,
```
Comment (laurelfulford) [inline newspack-theme/inc/color-patterns.php:307]: There's an extra comma here:

```suggestion
				.site-footer .widget-title,
				.site-footer .widgettitle,
```

**Review by laurelfulford — CHANGES_REQUESTED** [review]
Thanks for taking the time to make these updates, @hawkeye126! 

I did my best to review the diff, though I may have missed some things since there are changes that display that aren't actually included in this commit (just from updating your branch from the theme's updates).

If you have any questions at all, please let me know! Again, I also totally understand if this is a lot -- it's a confusing theme, and I've definitely gone through the same convoluted testing with my own commits, since there are so many weird cases you need to check for!

File: newspack-joseph/inc/child-typography.php [hunk 2]
```diff
@@ -74,12 +76,14 @@ function newspack_joseph_custom_typography_css() {
 	if ( true === get_theme_mod( 'accent_allcaps', true ) ) {
 		$css_blocks        .= '
 			.accent-header,
+			.widgettitle,
```
[the remark this answers is not in the document]
Comment (hawkeye126) [reply newspack-joseph/inc/child-typography.php]: Resolved here [707e90a](https://github.com/Automattic/newspack-theme/pull/1532/commits/707e90aae17aff3e05b200f205f6f9dd5bd6cc6c)

File: newspack-joseph/inc/child-typography.php [hunk 3]
```diff
@@ -54,6 +55,7 @@ function newspack_joseph_custom_typography_css() {
 			}
 			.block-editor-block-list__layout .block-editor-block-list__block .article-section-title,
 			.block-editor-block-list__layout .block-editor-block-list__block.accent-header,
+			.block-editor-block-list__layout .block-editor-block-list__block.widgettitle,
```
[the remark this answers is not in the document]
Comment (hawkeye126) [reply newspack-joseph/inc/child-typography.php]: Resolved here https://github.com/Automattic/newspack-theme/pull/1532/commits/2eb71118cda3e5745992b4d22f27c05611fde7d7

File: newspack-joseph/inc/child-typography.php [hunk 4]
```diff
@@ -74,12 +76,14 @@ function newspack_joseph_custom_typography_css() {
...
 		$css_blocks        .= '
 			.accent-header,
+			.widgettitle,
 			.article-section-title {
 				text-transform: uppercase;
 			}
 		';
 		$editor_css_blocks .= '
 			.block-editor-block-list__layout .block-editor-block-list__block.accent-header,
+			.block-editor-block-list__layout .block-editor-block-list__block.widgettitle,
```
[the remark this answers is not in the document]
Comment (hawkeye126) [reply newspack-joseph/inc/child-typography.php]: Resolved here https://github.com/Automattic/newspack-theme/pull/1532/commits/5c8175c7d24fa8d368f6cd088321016ca65bf63f

File: newspack-joseph/sass/style.scss [hunk 5]
```diff
@@ -125,7 +128,8 @@ div.wpnbha .article-section-title,
 
 .accent-header,
 .article-section-title,
-.widget-title.accent-header {
+.widget-title.accent-header,
+.widgettitle {
```
[the remark this answers is not in the document]
Comment (hawkeye126) [reply newspack-joseph/sass/style.scss]: Resolved here [5c8175c](https://github.com/Automattic/newspack-theme/pull/1532/commits/5c8175c7d24fa8d368f6cd088321016ca65bf63f)

File: newspack-joseph/sass/style.scss [hunk 6]
```diff
@@ -272,12 +294,14 @@ div.wpnbha .article-section-title,
 }
 
 .accent-header:not( .widget-title )::before,
+.widgettitle:not( .widget-title )::before,
```
[the remark this answers is not in the document]
Comment (hawkeye126) [reply newspack-joseph/sass/style.scss]: Resolved here [5c8175c](https://github.com/Automattic/newspack-theme/pull/1532/commits/5c8175c7d24fa8d368f6cd088321016ca65bf63f)

File: newspack-joseph/sass/style.scss [hunk 7]
```diff
@@ -272,12 +294,14 @@ div.wpnbha .article-section-title,
...
 .cat-links::before {
 	background: $color__text-main;
 }
 
 /* Author Bio */
-.author-bio .accent-header {
+.author-bio .accent-header,
+.author-bio .widgettitle {
```
[the remark this answers is not in the document]
Comment (hawkeye126) [reply newspack-joseph/sass/style.scss]: Resolved here [5c8175c](https://github.com/Automattic/newspack-theme/pull/1532/commits/5c8175c7d24fa8d368f6cd088321016ca65bf63f)

File: newspack-joseph/sass/style-editor.scss [hunk 8]
```diff
@@ -91,44 +91,42 @@ Newspack Joseph Editor Styles
 }
 
 /* Newspack Homepage Articles */
-.article-section-title {
+.article-section-title,
+.accent-header,
+.widgettitle {
```
[the remark this answers is not in the document]
Comment (hawkeye126) [reply newspack-joseph/sass/style-editor.scss]: Resolved here https://github.com/Automattic/newspack-theme/pull/1532/commits/f464f1de2c7722275c6d64e1cff9de4485c41a91

File: newspack-nelson/inc/child-typography.php [hunk 9]
```diff
@@ -37,19 +37,23 @@ function newspack_nelson_custom_typography_css() {
...
+			.site-footer .widget-title ,
+			.site-footer .widgettitle {
 				letter-spacing: 0.05em;
 				text-transform: uppercase;
 			}
 		';
 		$editor_css_blocks .= '
 			.block-editor-block-list__layout .block-editor-block-list__block.accent-header,
+			.block-editor-block-list__layout .block-editor-block-list__block.widget-title,
+			.block-editor-block-list__layout .block-editor-block-list__block.widgettitle,
```
[the remark this answers is not in the document]
Comment (hawkeye126) [reply newspack-nelson/inc/child-typography.php]: Resolve here [b040c930][https://github.com/Automattic/newspack-theme/pull/1532/commits/b040c93054a2c2bab7175e5601302c324a4935e7]

File: newspack-nelson/sass/style.scss [hunk 10]
```diff
@@ -261,7 +271,9 @@ div.wpnbha .article-section-title,
 }
 
 .accent-header,
+.widgettitle,
 .wp-block-columns .wp-block-column > .accent-header,
+.wp-block-columns .wp-block-column > .widgettitle,
```
[the remark this answers is not in the document]
Comment (hawkeye126) [reply newspack-nelson/sass/style.scss]: Resolved here 579c04fa

File: newspack-sacha/inc/child-color-patterns.php [hunk 11]
```diff
@@ -102,6 +105,7 @@ function newspack_sacha_custom_colors_css() {
 	$editor_css = '
 		.block-editor-block-list__layout .block-editor-block-list__block .entry-meta .byline a,
 		.block-editor-block-list__layout .block-editor-block-list__block.accent-header,
+		.block-editor-block-list__layout .block-editor-block-list__block.widgettitle,
```
[the remark this answers is not in the document]
Comment (hawkeye126) [reply newspack-sacha/inc/child-color-patterns.php]: Resolved here b0de3ae6

File: newspack-sacha/inc/child-typography.php [hunk 12]
```diff
@@ -42,12 +42,14 @@ function newspack_sacha_custom_typography_css() {
...
 		$css_blocks        .= '
 			.accent-header,
+			.widgettitle,
 			.article-section-title {
 				text-transform: uppercase;
 			}
 		';
 		$editor_css_blocks .= '
 			.block-editor-block-list__layout .block-editor-block-list__block.accent-header,
+			.block-editor-block-list__layout .block-editor-block-list__block.widgettitle,
```
[the remark this answers is not in the document]
Comment (hawkeye126) [reply newspack-sacha/inc/child-typography.php]: Resolved here 6c0f9a33

File: newspack-sacha/sass/style-editor.scss [hunk 13]
```diff
@@ -13,6 +13,7 @@ Newspack Sacha Editor Styles
 }
 
 .accent-header,
+.widgettitle,
```
[the remark this answers is not in the document]
Comment (hawkeye126) [reply newspack-sacha/sass/style-editor.scss]: Resolved here 5827df1b

File: newspack-sacha/sass/style.scss [hunk 14]
```diff
@@ -72,12 +72,14 @@ Nicolas Gallagher and Jonathan Neal https://necolas.github.io/normalize.css/
...
 // Accent headers
 .accent-header,
+.widgettitle,
 .article-section-title,
 .cat-links {
 	color: $color__primary;
 }
 
 .accent-header,
+.widgettitle,
```
[the remark this answers is not in the document]
Comment (hawkeye126) [reply newspack-sacha/sass/style.scss]: Resolved here 942bd624, please ignore 8aac9fde

File: newspack-sacha/sass/style.scss [hunk 15]
```diff
@@ -111,11 +114,13 @@ div.wpnbha .article-section-title,
 .mobile-sidebar,
 .site-footer {
 	.accent-header,
+	.widgettitle,
 	.article-section-title {
 		color: inherit;
 	}
 
 	.accent-header,
+	.widgettitle,
```
[the remark this answers is not in the document]
Comment (hawkeye126) [reply newspack-sacha/sass/style.scss]: Resolved here fa68423

File: newspack-sacha/sass/style.scss [hunk 16]
```diff
@@ -355,7 +360,8 @@ cite {
 	display: block;
 	text-align: center;
 
-	.accent-header {
+	.accent-header
+	.widgettitle, {
```
[the remark this answers is not in the document]
Comment (hawkeye126) [reply newspack-sacha/sass/style.scss]: Resolved here ecc6842c

File: newspack-scott/inc/child-color-patterns.php [hunk 17]
```diff
@@ -35,6 +35,7 @@ function newspack_scott_custom_colors_css() {
 
 	$theme_css = '
 		.accent-header:not(.widget-title):before,
+		.widgettitle:not(.widget-title):before,
```
[the remark this answers is not in the document]
Comment (hawkeye126) [reply newspack-scott/inc/child-color-patterns.php:38]: Resolved here de5112a9

File: newspack-scott/inc/child-color-patterns.php [hunk 18]
```diff
@@ -51,12 +52,14 @@ function newspack_scott_custom_colors_css() {
 		}
 
 		.mobile-sidebar .article-section-title,
-		.mobile-sidebar .accent-header {
+		.mobile-sidebar .accent-header,
+		.mobile-sidebar .widgettitle {
```
[the remark this answers is not in the document]
Comment (hawkeye126) [reply newspack-scott/inc/child-color-patterns.php]: Resolved here f7ed704

File: newspack-scott/inc/child-color-patterns.php [hunk 19]
```diff
@@ -51,12 +52,14 @@ function newspack_scott_custom_colors_css() {
...
 			color: ' . esc_html( $primary_color_contrast ) . ';
 		}
 
 		.mobile-sidebar .article-section-title::before,
-		.mobile-sidebar .accent-header::before {
+		.mobile-sidebar .accent-header::before,
+		.mobile-sidebar .widgetitle::before {
```
[the remark this answers is not in the document]
Comment (hawkeye126) [reply newspack-scott/inc/child-color-patterns.php:60]: Resolved here 52b092a7

File: newspack-scott/inc/child-color-patterns.php [hunk 20]
```diff
@@ -110,6 +113,7 @@ function newspack_scott_custom_colors_css() {
 			}
 
 			.site-footer .accent-header::before,
+			.site-footer .widgettitle::before,
 			.site-footer .article-section-title::before {
```
[the remark this answers is not in the document]
Comment (hawkeye126) [reply newspack-scott/inc/child-color-patterns.php:115]: Resolved here 3ac913c2

File: newspack-scott/inc/child-color-patterns.php [hunk 21]
```diff
@@ -120,6 +124,7 @@ function newspack_scott_custom_colors_css() {
 
 	$editor_css = '
 		.block-editor-block-list__layout .block-editor-block-list__block.accent-header:before,
+		.block-editor-block-list__layout .block-editor-block-list__block.widgettitle:before,
```
[the remark this answers is not in the document]
Comment (hawkeye126) [reply newspack-scott/inc/child-color-patterns.php:125]: Resolved here 32224247

File: newspack-joseph/sass/style.scss [hunk 22]
```diff
@@ -146,13 +150,31 @@ div.wpnbha .article-section-title,
...
+}
+
 .cat-links {
 	a {
 		color: inherit;
 	}
 }
 
 .accent-header:not( .widget-title ),
+.widgettitle:not( .widget-title ),
```
[the remark this answers is not in the document]
Comment (hawkeye126) [reply newspack-joseph/sass/style.scss]: Resolved here 5c8175c

File: newspack-scott/inc/child-typography.php [hunk 23]
```diff
@@ -37,6 +37,7 @@ function newspack_scott_custom_typography_css() {
 	if ( true === get_theme_mod( 'accent_allcaps', true ) ) {
 		$css_blocks        .= '
 			.accent-header:not(.widget-title),
+			.widgettitle:not(.widget-title),
 			.article-section-title,
```
[the remark this answers is not in the document]
Comment (hawkeye126) [reply newspack-scott/inc/child-typography.php]: Resolved here facd5ed2

File: newspack-scott/inc/child-typography.php [hunk 24]
```diff
@@ -47,6 +48,7 @@ function newspack_scott_custom_typography_css() {
 		';
 		$editor_css_blocks .= '
 			.block-editor-block-list__layout .block-editor-block-list__block.accent-header,
+			.block-editor-block-list__layout .block-editor-block-list__block.widgettitle,
```
[the remark this answers is not in the document]
Comment (hawkeye126) [reply newspack-scott/inc/child-typography.php:50]: Resolved here 73e2ef93

File: newspack-scott/sass/style-editor.scss [hunk 25]
```diff
@@ -8,6 +8,7 @@ Newspack Scott Editor Styles
 @import '../../newspack-theme/sass/style-editor-base';
 
 .accent-header,
+.widgettitle,
```
[the remark this answers is not in the document]
Comment (hawkeye126) [reply newspack-scott/sass/style-editor.scss:11]: Resolved here a37e812a

File: newspack-scott/sass/style.scss [hunk 26]
```diff
@@ -26,6 +26,7 @@ Nicolas Gallagher and Jonathan Neal https://necolas.github.io/normalize.css/
 /* Child theme-specific overrides */
 
 .accent-header:not( .widget-title ),
+.widgettitle:not( .widget-title ),
 .article-section-title,
```
[the remark this answers is not in the document]
Comment (hawkeye126) [reply newspack-scott/sass/style.scss:30]: Resolved here 27486889

File: newspack-scott/sass/style.scss [hunk 27]
```diff
@@ -48,7 +49,9 @@ Nicolas Gallagher and Jonathan Neal https://necolas.github.io/normalize.css/
 }
 
 .accent-header,
+.widgettitle,
 .wp-block-columns .wp-block-column > .accent-header,
+.wp-block-columns .wp-block-column > .widgettitle,
```
[the remark this answers is not in the document]
Comment (hawkeye126) [reply newspack-scott/sass/style.scss:54]: Resolved here 3249b214

File: newspack-scott/sass/style.scss [hunk 28]
```diff
@@ -118,7 +121,8 @@ div.wpnbha .article-section-title {
 
 /* Author Bio */
 
-.author-bio .accent-header {
+.author-bio .accent-header,
+.author-bio .widgettitle {
```
[the remark this answers is not in the document]
Comment (hawkeye126) [reply newspack-scott/sass/style.scss]: Resolved here 8b4633ee

File: newspack-scott/sass/style.scss [hunk 29]
```diff
@@ -294,7 +298,8 @@ div.wpnbha .article-section-title {
 .mobile-sidebar,
 .site-footer {
 	.article-section-title,
-	.accent-header {
+	.accent-header,
+	.widgettitle {
```
[the remark this answers is not in the document]
Comment (hawkeye126) [reply newspack-scott/sass/style.scss]: Resolved here 00c8e45a

File: newspack-theme/inc/color-patterns.php [hunk 30]
```diff
@@ -329,6 +330,7 @@ function newspack_custom_colors_css() {
 					background: ' . esc_html( newspack_adjust_brightness( $primary_color, -20 ) ) . ';
 				}
 				.mobile-sidebar .accent-header,
+				.mobile-sidebar .widgettitle,
 				.mobile-sidebar .article-section-title {
```
[the remark this answers is not in the document]
Comment (hawkeye126) [reply newspack-theme/inc/color-patterns.php]: Resolved here 585dd8da

File: newspack-theme/inc/color-patterns.php [hunk 31]
```diff
@@ -374,11 +376,13 @@ function newspack_custom_colors_css() {
 					}
 
 					.site-footer .accent-header,
+					.site-footer .widgettitle,
 					.site-footer .article-section-title {
```
[the remark this answers is not in the document]
Comment (hawkeye126) [reply newspack-theme/inc/color-patterns.php]: Resolved here 646ad272

File: newspack-theme/inc/color-patterns.php [hunk 32]
```diff
@@ -374,11 +376,13 @@ function newspack_custom_colors_css() {
...
 						border-color: ' . newspack_adjust_brightness( $footer_color, -20 ) . ';
 					}
 
 					.site-footer .accent-header,
+					.site-footer .widgettitle,
 					.site-footer .article-section-title {
```
[the remark this answers is not in the document]
Comment (hawkeye126) [reply newspack-theme/inc/color-patterns.php]: Resolved here bfc2d4bf

File: newspack-theme/inc/color-patterns.php [hunk 33]
```diff
@@ -602,7 +606,8 @@ function newspack_custom_colors_css() {
 		$editor_css .= '
 			.block-editor-block-list__layout .block-editor-block-list__block .entry-meta .byline a,
 			.block-editor-block-list__layout .block-editor-block-list__block .wp-block-newspack-blocks-homepage-articles:not(.has-text-color) .article-section-title,
-			.block-editor-block-list__layout .block-editor-block-list__block.accent-header {
+			.block-editor-block-list__layout .block-editor-block-list__block.accent-header,
+			.block-editor-block-list__layout .block-editor-block-list__block.widgettitle {
```
[the remark this answers is not in the document]
Comment (hawkeye126) [reply newspack-theme/inc/color-patterns.php]: Resolved here d162589a

File: newspack-theme/inc/typography.php [hunk 34]
```diff
@@ -387,12 +387,14 @@ function newspack_custom_typography_css() {
 		if ( ! is_child_theme() ) {
 			$css_blocks        .= '
 				.accent-header,
+				.widgettitle,
```
[the remark this answers is not in the document]
Comment (hawkeye126) [reply newspack-theme/inc/typography.php]: Resolved here 14518429

File: newspack-theme/inc/typography.php [hunk 35]
```diff
@@ -387,12 +387,14 @@ function newspack_custom_typography_css() {
...
 				.article-section-title {
 					text-transform: uppercase;
 				}
 			';
 			$editor_css_blocks .= '
 				.block-editor-block-list__layout .block-editor-block-list__block.accent-header,
+				.block-editor-block-list__layout .block-editor-block-list__block.widgettitle,
 				.block-editor-block-list__layout .block-editor-block-list__block .article-section-title {
```
[the remark this answers is not in the document]
Comment (hawkeye126) [reply newspack-theme/inc/typography.php]: Resolved here 29429181

File: newspack-theme/sass/blocks/_blocks.scss [hunk 36]
```diff
@@ -366,7 +366,8 @@ p.has-background {
 
 		&:first-child,
 		&:first-child .article-section-title, // when Load More is enabled.
-		&.accent-header + div.wpnbha {
+		&.accent-header + div.wpnbha,
+		&.widgettitle + div.wpnbha {
```
[the remark this answers is not in the document]
Comment (hawkeye126) [reply newspack-theme/sass/blocks/_blocks.scss]: Resolved here 658877b6

File: newspack-theme/sass/site/secondary/_widgets.scss [hunk 37]
```diff
@@ -141,11 +141,21 @@
...
 
+.widgettitle {
+	@extend .widget-title;
+	@extend .accent-header;
+
+	#secondary & {
+		text-transform: uppercase;
+	}
+}
+
```
[the remark this answers is not in the document]
Comment (hawkeye126) [reply newspack-theme/sass/site/secondary/_widgets.scss:158]: Thanks for the clarification @laurelfulford, I have removed the style here 658877b6

File: newspack-theme/sass/style-editor-base.scss [hunk 38]
```diff
@@ -251,6 +251,7 @@ figcaption,
 
 .wp-block-cover {
 	.accent-header,
+	.widgettitle,
 	.article-section-title {
```
[the remark this answers is not in the document]
Comment (hawkeye126) [reply newspack-theme/sass/style-editor-base.scss]: Resolved here b2361787

File: newspack-theme/sass/styles/style-default/style-default-editor.scss [hunk 39]
```diff
@@ -1,4 +1,5 @@
 .accent-header,
+.widgettitle,
 .article-section-title {
```
[the remark this answers is not in the document]
Comment (hawkeye126) [reply newspack-theme/sass/styles/style-default/style-default-editor.scss]: Resolved here 614b6c8c

File: newspack-theme/sass/styles/style-default/style-default.scss [hunk 40]
```diff
@@ -1,6 +1,7 @@
 /* Headers */
 
 .accent-header,
+.widgettitle,
```
[the remark this answers is not in the document]
Comment (hawkeye126) [reply newspack-theme/sass/styles/style-default/style-default.scss]: Resolved here 64808313

File: newspack-theme/sass/styles/style-default/style-default.scss [hunk 41]
```diff
@@ -9,7 +10,9 @@
 }
 
 .accent-header,
-.wp-block-columns .wp-block-column > .accent-header,
+.widgettitle,
+.wp-block-columns .wp-block-column > .accent-header,,
+.wp-block-columns .wp-block-column > .widgettitle,
```
[the remark this answers is not in the document]
Comment (hawkeye126) [reply newspack-theme/sass/styles/style-default/style-default.scss]: Resolved here 5169a736

File: newspack-theme/sass/typography/_headings.scss [hunk 42]
```diff
@@ -205,7 +205,8 @@ h6 {
 	text-transform: none;
 }
 
-.accent-header {
+.accent-header,
+.widgettitle {
```
[the remark this answers is not in the document]
Comment (hawkeye126) [reply newspack-theme/sass/typography/_headings.scss]: Resolved here ed027f03

File: newspack-theme/inc/color-patterns.php (hunk 1 above)
Comment (hawkeye126) [reply newspack-theme/inc/color-patterns.php]: Resolved here b58425d8

**Comment by hawkeye126** [comment]
@laurelfulford Thanks for the review and explanations - it's very helpful 💯  !  I have addressed everything (I think anyway) and am ready for another review.  Sorry for the amount of commits, I broke out the fixes to hopefully make it easier on you to review.  If you'd like me to squash them and do some branch cleanup please let me know.

**Comment by hawkeye126** [comment]
@laurelfulford I did some branch cleanup to reduce commits and better comply with the contribution guidelines.  Please let me know if you'd like to use that branch, I will update the PR to be against the Automattic repo (right now it's against my fork).  Thanks!

https://github.com/hawkeye126/newspack-theme/pull/12

**Comment by laurelfulford** [comment]
Thanks @hawkeye126! Apologies for my really late follow up here -- I took some time off for the holidays, and then had to tackle some WordPress 5.9 testing before getting back into things! :) 

> I did some branch cleanup to reduce commits and better comply with the contribution guidelines. Please let me know if you'd like to use that branch, I will update the PR to be against the Automattic repo (right now it's against my fork).

That would be amazing if it's not too much trouble! 🙌 Thank you!

**Comment by hawkeye126** [comment]
@laurelfulford here's the updated PR, please let me know if there is anything more I can address.  https://github.com/Automattic/newspack-theme/pull/1678

**Comment by laurelfulford** [comment]
Awesome, thanks @9ete! 🙌 I'm going to close this one for now and will give feedback in the new PR, but we can reopen this if needed for any reason!

**Pull request closed without merging**